app/models/order_products.go

```diff
@@ -0,0 +1,7 @@
+package models
+
+type OrderProducts struct {
+	OrderID   string `gorm:"column:order_id;primaryKey"`
```

Comment:
อย่าไปใส่เป็น primary key ดิ table นี้มันเป็น many to many

Reply:
finish i change to not null